pkg/engine/templates_generated.go

```diff
@@ -43148,13 +43153,21 @@ function Install-Containerd {
     [Parameter(Mandatory = $true)][string]
     $CNIConfDir
   )
+
+  $svc = Get-Service -Name containerd -ErrorAction SilentlyContinue
```

Comment:
This bit is needed for work containerd VHD work I'm also working on (but i'll highlight it anyway)